configurations/default/server.yml.tmp

```diff
@@ -1,27 +1,60 @@
 application:
-  assets_bucket: datatools-staging # dist directory
```

Comment:
These are all added so that the e2e tests can work on a local machine. I believe the config was updated to reflect one of our staging configs, so that's probably where I got some of the weird values.